terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf

```diff
@@ -178,10 +178,9 @@ module "server_containerized" {
   }
   main_disk_size = 3000
   runtime = "podman"
-// WORKAROUND
-// We want to test the most recent code right now until the submissions for Beta 2 are ready
-//  container_repository = "registry.suse.de/suse/sle-15-sp6/update/products/manager50/containerfile/suse/manager/5.0/x86_64"
-  container_repository = "registry.suse.de/devel/galaxy/manager/head/containerfile/suse/manager/5.0/x86_64"
+  container_repository = "registry.suse.de/suse/sle-15-sp6/update/products/manager50/containerfile/suse/manager/5.0/x86_64"
+  // Most recent code. Enable again once Beta 2 will be approved
```

Comment:
If this remark applies to next line, please end it with `:` so that becomes clear